packages/web-components/src/appointment-planner/client/styles/appointment-planner.styles.ts

```diff
@@ -0,0 +1,14 @@
+import { css } from 'emotion'
+
+export const apointmentPlannerSvg = css`
+  path {
+    fill: #c7c7c7;
```

Comment:
Colors, px

Reply:
@willmcvay I don't understand this review comment. Could you help to to clarify it? Thanks

Reply:
Yeah, sorry, I wasn't clear - was the same point as before - all colors should be themeable and limit the use of px